src/Sulu/Bundle/MediaBundle/Controller/MediaController.php

```diff
@@ -322,11 +322,10 @@ private function getListRepresentation(Request $request)
      * @deprecated
      *
      * @param FieldDescriptorInterface[] $fieldDescriptors
-     * @param array $types
      *
      * @return DoctrineListBuilder
      */
-    private function getListBuilder(Request $request, array $fieldDescriptors, $types)
+    private function getListBuilder(Request $request, array $fieldDescriptors, array $types)
```

Comment:
Would crash later in line 360 when using `$listbuilder->in` as the second parameter has array parameter type.